### ISSUE TITLE
Rule proposal: `prefer-math-abs`

### ISSUE THREAD

**Issue by fisker** [sindresorhus/eslint-plugin-unicorn#1249]
## Fail

```js
const a = b < 0 ? -b : b;
const a = b <= 0 ? -b : b;
const a = b < 0 ? 0 - b : b;
const a = b <= 0 ? 0 - b : b;
```

```js
const a = b > 0 ? b : -b;
const a = b >= 0 ? b : -b;
```

```js
const a = b < 0 ? b * -1 : b;
```

```js
if (number > POSITIVE_CONSTANT || number < -POSITIVE_CONSTANT) {}
```

## Pass

```js
const a = Math.abs(b);
```

```js
if (Math.abs(number) > POSITIVE_CONSTANT) {}
```

**Comment by sindresorhus** [comment]
This is now accepted.

**Comment by sindresorhus** [comment]
Added in https://github.com/sindresorhus/eslint-plugin-unicorn/commit/7df5b485fa7d03959257a7033cf7b2687651e3ad
